izug/refegovservice/profiles/default/types/izug.refegovservice.egovservice.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    name="izug.refegovservice.egovservice"
+    meta_type="Dexterity FTI"
+    i18n:domain="izug.refegovservice">
+
+  <!-- Basic properties -->
+  <property
+      i18n:translate=""
+      name="title">EGov Service</property>
+  <property
+      i18n:translate=""
+      name="description">Description of an Government Service</property>
+
+  <property name="allow_discussion">False</property>
+  <property name="factory">izug.refegovservice.egovservice</property>
+  <property name="icon_expr">string:${portal_url}/document_icon.png</property>
+  <property name="link_target"></property>
+
+  <!-- Hierarchy control -->
+  <property name="global_allow">True</property>
+  <!-- Schema, class and security -->
+  <property name="add_permission">izug.refegovservice.AddEGovService</property>
+  <property name="klass">izug.refegovservice.content.e_gov_service.EGovService</property>
+  <property name="schema">izug.refegovservice.content.e_gov_service.IEGovService</property>
+
+  <!-- Enabled behaviors -->
+  <property name="behaviors" purge="false">
+
+    <element value="plone.app.content.interfaces.INameFromTitle"/>
+    <element value="plone.app.dexterity.behaviors.metadata.IBasic"/>
+    <!-- categorization includes the language field -->
+    <element value="plone.app.dexterity.behaviors.metadata.ICategorization"/>
+    <!-- TODO - are some of the below unnecessary? -->
+    <element value="plone.app.dexterity.behaviors.metadata.IPublication"/>
+    <element value="plone.app.dexterity.behaviors.metadata.IOwnership"/>
+    <element value="plone.app.dexterity.behaviors.exclfromnav.IExcludeFromNavigation"/>
+
+    <!-- <element value="plone.app.referenceablebehavior.referenceable.IReferenceable" /> -->
```

Comment:
I think this should be enabled so that reference work properly.

Reply:
@jone, but are not all contenttypes moving to Dexterity? (or do we need it during the transition?)

Reply:
removed as agreed